package.json

```diff
@@ -31,6 +31,8 @@
     "react-native": "zulip/react-native#master",
     "react-native-device-info": "^0.9.7",
     "react-native-drawer": "^2.3.0",
+    "react-native-menu": "^0.20.2",
+    "react-native-modalbox": "^1.3.9",
```

Comment:
Can we use the built-in modal that comes with React native? https://facebook.github.io/react-native/docs/modal.html

Or actually, better yet, we can push a new screen onto the Navigator